tests/src/main/java/net/neoforged/neoforge/oldtest/client/rendering/RenderBuffersTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.oldtest.client.rendering;
```

Comment:
This is still the legacy testing system. This should probably use the new system. @Matyrobbrt anything specific for client-side events?

Reply:
Since they're manual, they should have the event as part of the test method. See https://github.com/neoforged/NeoForge/blob/1.20.x/tests/src/main/java/net/neoforged/neoforge/debug/client/ClientEventTests.java. If that's not possible, the normal way is still fine.

Reply:
Does it needs to be something visual to see the effect and confirm? I'm willing to convert it to the new test framework system, but I'm not sure how to test this kind of stuff, because it's simply just adding BufferBuilder's to the vanilla map

Reply:
It's also hard to test visually because in the simplest cases there is no difference between registering the buffer and not registering it.

Reply:
The most obvious difference between registering and not registering the buffer will be a crash in the `VertexMultiConsumer` case I mentioned.
Moving the two handlers from the test mod to a proper test and marking the test as either succesful or failed depending on the map entry being present in the second event should do the trick. Requesting user confirmation doesn't make much sense for this specific case in my opinion.